assets/js/shortcake-bakery.js

```diff
@@ -30,8 +30,21 @@
 				parentWidth = parent.innerWidth;
 			}
 
-			var trueHeight = el.data('true-height') ? el.data('true-height') : 360;
-			var trueWidth = el.data('true-width') ? el.data('true-width') : 640;
+			var trueHeight = 360,
+				trueWidth = 640;
+
+			if ( el.data('true-height') ) {
+				trueHeight = el.data('true-height').replace( 'px', '' );
+			} else if ( el.attr('height') ) {
+				trueHeight = el.attr('height').replace( 'px', '' );
+			}
+
+			if ( el.data('true-width') ) {
+				trueWidth = el.data('true-width').replace( 'px', '' );
+			} else if ( el.attr('width') ) {
+				trueWidth = el.attr('width').replace( 'px', '' );
```

Comment:
> What happens when width is set to a value like 100%?

Good question. It would break as I wrote the code.

> Does that need to be handled specially?

Probably. Do you have a suggested implementation?

I suppose the alternative to this pull request is that we should be adding `data-true-width` and `data-true-height` to all elements that include `shortcake-bakery-responsive` class.